core/src/main/java/hudson/search/Search.java

```diff
@@ -68,10 +66,6 @@
  * @see SearchableModelObject
  */
 public class Search implements StaplerProxy {
-    @Restricted(NoExternalUse.class) // used from stapler views only
-    public static String encodeQuery(String query) throws UnsupportedEncodingException {
```

Comment:
Not sure whether it's safe to remove it

Reply:
I would say it is because of `NoExternalUse`, and a Github search in the jenkinsci org doesn't reveal more usage than what I've replaced.  But let's ask @stephenc, who introduced this method in the first place, in 5d920577484.

Reply:
It's `@Restricted` and has been from the beginning. If anyone is using it, that's a bug because they've messed with their tooling. That's why that annotation exists.